modules/actions/workflows.go

```diff
@@ -255,9 +255,9 @@ func matchPushEvent(commit *git.Commit, pushPayload *api.PushPayload, evt *jobpa
 				matchTimes++
 			}
 		case "paths":
-			filesChanged, err := commit.GetFilesChangedSinceCommit(pushPayload.Before)
+			filesChanged, err := commit.GetFilesChanged()
```

Comment:
I think you need to check whether `pushPayload.Before` is empty, if it's empty use `commit.GetFilesChanged()`, otherwise use the old logic.

Reply:
fixed in cfa1815fa49be5461c77485462782ea4d4421c66